src/journal/journalctl.c

```diff
@@ -2661,14 +2692,23 @@ int main(int argc, char *argv[]) {
                         if (n_shown == 0 && !arg_quiet)
                                 printf("-- No entries --\n");
 
-                        if (arg_show_cursor) {
+                        if (arg_show_cursor || arg_cursor_file) {
                                 _cleanup_free_ char *cursor = NULL;
 
                                 r = sd_journal_get_cursor(j, &cursor);
                                 if (r < 0 && r != -EADDRNOTAVAIL)
                                         log_error_errno(r, "Failed to get cursor: %m");
-                                else if (r >= 0)
-                                        printf("-- cursor: %s\n", cursor);
+                                else if (r >= 0) {
+                                        if (arg_cursor_file) {
+                                                r = write_string_file(arg_cursor_file, cursor,
+                                                    WRITE_STRING_FILE_CREATE | WRITE_STRING_FILE_ATOMIC);
+                                                if (r < 0)
+                                                        log_error_errno(r,
+                                                            "Failed to write new cursor to %s: %m",
+                                                            arg_cursor_file);
```

Comment:
Spurious indentation.

Reply:
There's no setting for `arglist-cont-nonempty` in *.dir-locals.el*. What's the right setting?

Reply:
hmm, apparently the default of emacs appears to match our usual codebase then. we usually indent the first line broken argument to the same column as the first argument that is not line-broken. And if the first argument is already line broken, then it is indented to 8ch... If you want to turn that into a dir-locals line, by all means send a patch ;-)

Reply:
```
diff --git i/.dir-locals.el w/.dir-locals.el
index 676d755231..815afe172a 100644
--- i/.dir-locals.el
+++ w/.dir-locals.el
@@ -15,7 +15,8 @@
             (eval . (c-set-offset 'statement-case-open 0))
             (eval . (c-set-offset 'case-label 0))
             (eval . (c-set-offset 'arglist-intro '++))
-            (eval . (c-set-offset 'arglist-close 0))))
+            (eval . (c-set-offset 'arglist-close 0))
+            (eval . (c-set-offset 'arglist-cont-nonempty '(c-lineup-gcc-asm-reg c-lineup-arglist)))))
  (nxml-mode . ((nxml-child-indent . 2)
                (fill-column . 109)))
  (meson-mode . ((meson-indent-basic . 8)))
```

That's it. Can I include this in this PR?

Reply:
probably better in a separate pr